service/history/workflow/mutable_state_impl.go

```diff
@@ -3292,6 +3370,7 @@ func (ms *MutableStateImpl) ReplicateWorkflowExecutionUpdateAcceptedEvent(
 	ms.updateUpdateInfos[updateID] = &ui
 	if !existed {
 		ms.executionInfo.UpdateCount++
+		ms.approximateSize += ui.Size()
```

Comment:
also need to handle the case of `existed`?

Reply:
I think we are only updating one ID field, so the size shouldn't change if it already existed